addons/resource.language.en_gb/resources/strings.po

```diff
@@ -1289,6 +1292,9 @@ msgctxt "#297"
 msgid "Audio offset"
 msgstr ""
 
+#: addons/skin.confluence/720p/VideoOSD.xml
+#: skin.confluence/720p/VideoOSDBookmarks.xml
```

Comment:
I cant believe the amount of paint required for this path thing... Ill remove addons sigh.